todoman/cli.py

```diff
@@ -97,6 +102,35 @@ def command_wrap(*a, **kw):
     return command_wrap
 
 
+class AppContext:
+    def __init__(self):
+        self.config = None
+        self.db = None
+        self.porcelain = False
+
+    @cached_property
```

Comment:
Sorry, the `formatter` attribute below slipped completely past me. I blame the lack of sleep.